### PR TITLE
Post about tutorials contributed by students of a GRASS workshop

### PR DESCRIPTION
**Pull request by veroandreo**
This is a post that I'd like to publish as a GRASS news in which I highlight a couple of really nice tutorials contributed by students from a GRASS workshop I taught. Here the screenshots of how the news looks like:

The experience of asking students to develop tutorials as final exam of the GRASS workshop was really rewarding and I'd like to encourage others involved in teaching to write similar posts. I believe it's motivating for students to know their work will be recognized and shown within the software channels.

### CODE REVIEW

**Comment by hellik** [comment]
Looks good

**Review by neteler — APPROVED** [review]
Very nice!

File: content/news/2021_04_23_new_tutorials_made_by_students.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Topics chosen by the students were diverse and interesting: from changes 
+in snow cover in southern Argentina, to productivity of high altitude
+grasslands, wildfire simulations, segmentation to aid digitalization
+of implanted forests, network analysis, comparison of classification
+approaches to map urban areas, landscape characterization, 
+urban heat islands, spatial and temporal gap-filling and 
+species identification through OBIA and machine learning. Students
+overcame installation difficulties, learnt and studied new modules, 
+searched for data, explored different solutions. Some, even moved to
+Linux and learnt to use git. It was really satisfying to see their
```
Comment (wenzeslaus) [inline content/news/2021_04_23_new_tutorials_made_by_students.md:36]:
```suggestion
Linux and learnt to use Git. It was really satisfying to see their 
```

File: content/news/2021_04_23_new_tutorials_made_by_students.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+overcame installation difficulties, learnt and studied new modules, 
+searched for data, explored different solutions. Some, even moved to
+Linux and learnt to use git. It was really satisfying to see their 
+learning process and outcome.
+
+Here, I'd like to highlight two outstanding tutorials:
+
+- [​Wildfire Modeling in Yosemite National Park](https://github.com/acoiman/wildfire_modeling) by 
+[Abraham Coiman](https://acoiman.github.io/) from Venezuela. 
+In this tutorial, Abraham combines the use of GEE, geemap, geopandas and GRASS GIS
```
Comment (wenzeslaus) [inline content/news/2021_04_23_new_tutorials_made_by_students.md:43]:
```suggestion
In this tutorial, Abraham combines the use of GEE, geemap, GeoPandas and GRASS GIS
```

**Review by wenzeslaus — APPROVED** [review]

**Pull request merged**